src/main/scala/no/ndla/imageapi/controller/ImageController.scala

```diff
@@ -89,7 +113,28 @@ trait ImageController {
       val imageId = long("image_id")
       imageRepository.withId(imageId) match {
         case Some(image) => converterService.asApiImageMetaInformationWithApplicationUrl(image)
-        case None => halt(status = 404, body = Error(NOT_FOUND, s"Image with id $imageId not found"))
+        case None => halt(status = 404, body = Error(Error.NOT_FOUND, s"Image with id $imageId not found"))
+      }
+    }
+
+    post("/") {
+      val newImage = params.get("metadata")
+        .map(extract[NewImageMetaInformation])
+        .getOrElse(throw new ValidationException(errors=Seq(ValidationMessage("metadata", "The request must contain audio metadata"))))
+      val file = fileParams.getOrElse("file", throw new ValidationException(errors=Seq(ValidationMessage("file", "The request must contain one or more files"))))
```

Comment:
Hvordan er det løst i audio-api ift. å matche metadata mot filer? Eller er det en "convenience"-greie for å kunne bruke like metadata på mange filer?